src/main/java/com/flab/bbt/deal/controller/DealController.java

```diff
@@ -40,4 +42,13 @@ public CommonResponse getDeal(@PathVariable long id) {
         Deal deal = dealService.findDealById(id);
         return CommonResponse.success(deal);
     }
+
+    @PatchMapping("/{dealId}")
```

Comment:
(minor)
다른api들 처럼 {id}로 충분합니다.
```suggestion
    @PatchMapping("/{id}")
```

Reply:
수정했습니다 :)